packages/ember-views/lib/views/core_view.js

```diff
@@ -7,9 +7,9 @@ import ActionHandler, { deprecateUnderscoreActions } from 'ember-runtime/mixins/
 import { typeOf } from 'ember-runtime/utils';
 
 import { InteractiveRenderer } from 'ember-htmlbars/renderer';
+import DOMHelper from 'ember-htmlbars/system/dom-helper';
```

Comment:
@krisselden I was trying to make it so this could use `ember-htmlbars/system/dom-helper` or `ember-glimmer/dom` as necessary in order to make tests pass, but then I talked to @chadhietala regarding these. He said "Don’t port [views] They will be deleted", so I need to revise this.

Should the tests (outside of the ember-views package) that create a view in order to test something about helpers be updated to create a component instead?